api/migrations/20181115190951_update-previous-activities.js

```diff
@@ -0,0 +1,103 @@
+exports.up = async knex => {
+  await knex.schema.alterTable('apd_previous_activity_expenses', table => {
```

Comment:
Adds new columns, computes their values from the existing columns, and then removes the old columns.